plugins/modules/ldap_entry.py

```diff
@@ -36,6 +36,11 @@
       - If I(state=present), attributes necessary to create an entry. Existing
         entries are never modified. To assert specific attribute values on an
         existing entry, use M(community.general.ldap_attrs) module instead.
+      - Each attribute value can be a string for single-valued attributes or
+        a list of strings for multi-valued attributes.
+        To break long string values
+        into multiple lines for readability use the YAML block modifiers as
+        seen in the examples.
```

Comment:
```suggestion
      - If you specify values for this option in YAML, please note that you can improve
        readability for long string values by using YAML block modifiers as seen in the
        examples for this module.
      - Note that when using values that YAML/ansible-core interprets as other types,
        like C(yes), C(no) (booleans), or C(2.10) (float), make sure to quote them if
        these are meant to be strings. Otherwise you risk that the wrong values are
        sent to LDAP.
```